src/screens/AllVoucherScreen/AllVoucherScreen.tsx

```
@@ -70,21 +70,21 @@ const AllVoucherScreen: React.FC<Props> = ({ _voucherName, companyVoucherVersion
 
         console.log('-----GET ALL VOUCHERS-----', date.startDate, date.endDate)
         
         try {
-            const response = await CommonService.getAllVouchers(voucherName.toLocaleLowerCase(), date.startDate, date.endDate, ( page ?? pageCount.page ) + 1, pageCount.count, companyVoucherVersion, payload)
+            const response = await CommonService.getAllVouchers(voucherName.toLocaleLowerCase(), date.startDate, date.endDate,  page  , pageCount.count, companyVoucherVersion, payload)
             if (response?.status === 'success' && Array.isArray(response?.body?.items)) {
-                setVoucherData((prevVoucherData) => page === 0 ? response?.body?.items : [...prevVoucherData, ...response?.body?.items])
-                setPageCount((prevPageCount) => ({ page: response?.body?.items?.length < 25 ? -1 : ( page ?? prevPageCount.page ) + 1, count: prevPageCount.count }))
-                if (response?.body?.items?.length < 25) {
-                    setIsLoadingMore(false)
+                setVoucherData((prevVoucherData) => [...prevVoucherData, ...response?.body?.items])
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)